spec/api_auth_spec.rb

```diff
@@ -94,6 +112,24 @@ def hmac(secret_key, request)
       request['date'] = "٢٠١٤-٠٩-٠٨ ١٦:٣١:١٤ +٠٣٠٠"
       expect(ApiAuth.authentic?(request, "123")).to eq false
     end
+
+    context "canonical string contains the http_method" do
```

Comment:
Sure, I guess it can't hurt. To me that's more of a test that the canonical string is getting generated correctly, but still good to know that it's working as expected there.